app/src/main/java/com/lagradost/cloudstream3/ui/home/HomeFragment.kt

```diff
@@ -210,7 +213,13 @@ class HomeFragment : Fragment() {
             recycle.adapter = SearchAdapter(item.list.toMutableList(), recycle) { callback ->
                 handleSearchClickCallback(this, callback)
                 if (callback.action == SEARCH_ACTION_LOAD || callback.action == SEARCH_ACTION_PLAY_FILE) {
-                    bottomSheetDialogBuilder.dismissSafe(this)
+//                    bottomSheetDialogBuilder.dismissSafe(this)
+                    handler.postDelayed({
```

Comment:
why delayed and why `it.window?.setWindowAnimations(-1)` ?
`bottomSheetDialogBuilder.hide()` does the job just fine

Reply:
After clicking, the dialog will be hidden, and the new page will not be displayed immediately